product/views.py

```diff
@@ -1,3 +1,70 @@
-from django.shortcuts import render
+import json
+import re
+from django.http             import JsonResponse
+from django.views            import View
 
-# Create your views here.
+
+from product.models          import Product, Sale
+from user.models             import User
+from decorators.utils        import login_required
+
+import operator
+
+class ProductListView(View):  
+    # @login_required
+    def get(self,request, catCd, sorting):
+        try:
+            items_per_page = 16
+            sort_type = {
+                'low_price':'price',
+                'high_price':'-price',
+                'new':'create_at'
+                }
+            if catCd ==0:
```

Comment:
`catCd == 0`이 의미하는게 무엇인지 알기가 어렵습니다.
```suggestion
            if catCd ==0:
```

Reply:
전체를 선택할 경우 카테고리 상관없이 모든 상품을 필터링하기 위해 넣어 놨습니다.